datamodel.graphql

```diff
@@ -112,6 +112,20 @@ type ClassLevel {
     classRooms: [Classroom]
 }
 
+type TimeTable {
+    id: ID! @id
+    class: Classroom!
+    teacher: User
+    weekday: WeekDays!
+    starttime: String
+    endtime: String
+    type: TimetableType!
+    date: DateTime
+    subject: ClassSubject!
```

Comment:
some events don't have a subject. Eg Short breaks, lunch, Games time, Prayers day, visiting day, Academic day. We can change from subject to Event name.

Reply:
Let us have both subject and event name but as optional params

Reply:
okay